catalog/plugin/catalog-plugin-query-monitor-commands/src/main/java/ddf.catalog.querymonitorcommands/QueryMonitorCommandPrintTable.java

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.querymonitorcommands;
+
+import java.io.PrintStream;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.UUID;
+
+import org.apache.karaf.shell.api.action.Action;
+import org.apache.karaf.shell.api.action.Command;
+import org.fusesource.jansi.Ansi;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.querymonitor.ActiveSearch;
+import ddf.catalog.querymonitor.QueryMonitorPlugin;
+
+@Command(scope = "catalog", name = "printtable", description = "Sets flag to determine whether to delete active searches from table once they complete")
+public class QueryMonitorCommandPrintTable implements Action {
+
+    private static final Logger LOGGER =
+            LoggerFactory.getLogger(QueryMonitorCommandPrintTable.class);
+
+    QueryMonitorPlugin queryMonitor1 = null;
+
+    private static final int USER_MAX_LENGTH = 20;
+
+    private static final int SOURCE_MAX_LENGTH = 20;
+
+    private static final int SEARCH_MAX_LENGTH = 33;
+
+    private static final int TIME_MAX_LENGTH = 30;
+
+    private static final int UUID_MAX_LENGTH = 37;
+
+    private String formatString =
+            "%1$-" + USER_MAX_LENGTH + "s %2$-" + SOURCE_MAX_LENGTH + "s %3$-" + SEARCH_MAX_LENGTH
+                    + "s %4$-" + TIME_MAX_LENGTH + "s %5$-" + UUID_MAX_LENGTH + "s %n";
+
+    PrintStream console = null;
```

Comment:
We have an abstract class you can use to give you some of the command stuff and you can get rid of this boilerplate code `org.codice.ddf.commands.catalog.CommandSupport`